configs/webpack/base.coffee

```diff
@@ -181,6 +183,12 @@ makeDevelopmentBase = (projectConfig) ->
   publicPath = "#{servePath}/dist/"
   outputPath = "#{projectConfig.basePath}/"
 
+  if projectConfig.webpackDashboardSocketPort?
+    dashboardConfig =
+      port: projectConfig.webpackDashboardSocketPort
+
+  console.info("HALLO", dashboardConfig)
```

Comment:
Seems like we should elevate the fail-Travis-when-console-messages-are-present ; )